vendor.go

```diff
@@ -168,11 +181,13 @@ func closeFile(f io.Closer, name string) {
 func copyFile(src string, dst string) {
 	in, err := os.Open(src)
 	abortonerr(err, fmt.Sprintf("opening %s", src))
-	addCleanup(func() { closeFile(in, src) })
+
+	defer closeFile(in, src)
```

Comment:
It is beyond me why I used this whole addCleanup thing...I have no idea...right now it makes no sense :-(